bulwark/checks.py

```diff
@@ -60,32 +60,68 @@ def has_columns(df, columns, exact_cols=False, exact_order=False):
     return df
 
 
-def has_no_nans(df, columns=None):
-    """Asserts that there are no np.nans in `df`.
+def has_no_x(df, values=None, columns=None):
+    """Asserts that there are no user specified values in `df`.
 
     Args:
         df (pd.DataFrame): Any pd.DataFrame.
+        values (list): A list of values to check for in the pd.DataFrame.
         columns (list): A subset of columns to check for np.nans.
```

Comment:
Should read:

`A subset of columns to check for `values`.`

Reply:
Updated!